src/include/container/hash/linear_probe_hash_table.h

```diff
@@ -38,31 +39,36 @@ class LinearProbeHashTable {
    * @param buffer_pool_manager buffer pool manager to be used
    * @param comparator comparator for keys
    * @param num_buckets initial number of buckets contained by this hash table
+   * @param hash_fn
```

Comment:
Maybe tell them that they can use a pointer to murmur hash, either here or in the writeup.